extension/src/extension.ts

```diff
@@ -78,6 +78,8 @@ import {
     throwIfAbortError,
 } from './util'
 
+const path = require('path-browserify')
```

Comment:
Why this and not `import * as path from 'path'`?

Reply:
Isn't `path` a Node.js-specific import? I tried `import * as path from 'path'` but that threw an error when initializing the web worker.

Reply:
Yes, but it should pick a browser-compatible shim. Which error did you see?

In any case, we should never use `require()`, because the type of `path` here is now `any`, which is not good.

Reply:
I just tested this with `import * as path from 'path'` and did not get any error.

Reply:
I tested again, too, and did not get any error either. I'm not sure what went wrong last time 😕

👍 to switching.

Reply:
Last time, the Parcel cache on my machine might have been corrupt. I've seen the same error since and `rm -rf .cache/` fixes it. This usually happens when I `yarn link`.